src/init.sh

```diff
@@ -69,17 +74,27 @@ function init_kw()
     exit 2 # ENOENT
   fi
 
-  mkdir --parents "${PWD}/${KW_DIR}"
-  cp "$config_file_template" "${PWD}/${KW_DIR}/${name}"
-  cp "$vm_config_file_template" "${PWD}/${KW_DIR}/${vm_name}"
-  cp "$build_config_file_template" "${PWD}/${KW_DIR}/${build_name}"
-  cp "$deploy_config_file_template" "${PWD}/${KW_DIR}/${deploy_name}"
-  cp "$mail_config_file_template" "${PWD}/${KW_DIR}/${mail_name}"
-  cp "$notification_config_file_template" "${PWD}/${KW_DIR}/${notification_name}"
-  cp "$remote_file_template" "${PWD}/${KW_DIR}/${remote_name}"
-
-  sed --in-place --expression "s/USERKW/${USER}/g" -e '/^#?.*/d' "${PWD}/${KW_DIR}/${vm_name}"
-  sed --in-place --expression "s,SOUNDPATH,${KW_SOUND_DIR},g" -e '/^#?.*/d' "${PWD}/${KW_DIR}/${notification_name}"
+  cmd="mkdir --parents \"${PWD}/${KW_DIR}\""
```

Comment:
I don't think you need to add `"` inside `"`. In other words, drop the internal quotes for command and the other.